sensor_msgs/Image cam1
sensor_msgs/Image cam2
